kuma/javascript/src/document.jsx

```diff
@@ -97,12 +98,13 @@ const styles = {
         maxWidth: 1400,
         margin: '0 auto',
         gridTemplateColumns: '25% 75%',
-        gridTemplateAreas: '"side main"',
+        grinTemplateRows: 'minmax(200px, 1fr) 75%',
```

Comment:
Is this a typo? "grin" instead of "grid"?

Reply:
Oh boy 😄, it is indeed a typo. Thanks @MarcoZehe